settings.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+	// Gets URL of homepage
+	function homePageURL() {
+		$pageURL = 'http';
+		if (isset($_SERVER["HTTPS"])) 
+			$pageURL .= "s";
+		
+		$pageURL .= "://";
+		$pageURL .= $_SERVER["SERVER_NAME"];
+
+		return $pageURL;
+	}
+
+	// Get the inputs from the B2C Settings Page
+	$config_elements = get_option('b2c_config_elements');
+	
+	// Parse the settings entered in by the admin on the b2c settings page
+	$tenant = $config_elements['b2c_aad_tenant'];
+	$clientID = $config_elements['b2c_client_id'];
+	$generic_policy = $config_elements['b2c_subscriber_policy_id'];
+	$admin_policy = $config_elements['b2c_admin_policy_id'];
+	$edit_profile_policy = $config_elements['b2c_edit_profile_policy_id'];
+	$redirect_uri = urlencode(homePageURL()); 
+	if ($config_elements['b2c_verify_tokens']) $verify_tokens = 1;
+	else $verify_tokens = 0;
```

Comment:
Created a Settings class.

---
In reply to: [71235821](https://github.com/AzureAD/active-directory-b2c-wordpress-plugin-openidconnect/pull/3#discussion_r71235821) [](ancestors = 71235821,71196588)